packages/helpers/index.ts

```diff
@@ -0,0 +1,20 @@
+
+export * from './src/binaryFormat';
+export * from './src/constants';
+export * from './src/fast-sha256';
+export * from './src/input-helpers';
+export * from './src/merkle';
+export * from './src/poseidonHash';
+export * from './src/rsa';
+export * from './src/shaHash';
+export * from './src/sshFormat';
+export * from './src/uncompress';
+export * from './src/binaryFormat';
+export * from './src/dkim/index';
+export * from './src/zkp';
+
```

Comment:
remove empty lines at the end of file. we should have a linter hook automatically do this for you